sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Filtering/DerivedMetric.cs

```diff
@@ -110,7 +110,7 @@ public double Project(TTelemetry document)
 
         public override string ToString()
         {
-            return this.info.ToString();
+            return this.info.ToString()!;
```

Comment:
`this.info` is readonly and initialized in the ctor. It's possible that this could fail, but then it's unlikely that `ToString` would be invoked. I think it's fine to use `!` here.